contracts/misc/MarketView.sol

```diff
@@ -262,8 +262,16 @@ contract MarketView {
         RealityETH_v3_0 realitio = RealityETH_v3_0(market.realitio());
         uint256 _numberOfQuestions = numberOfQuestions(market);
         for (uint256 i = 0; i < _numberOfQuestions; i++) {
-            if (realitio.getFinalizeTS(market.questionIDs(i)) > 0) {
-                count += 1;
+            bytes32 question_id = market.questionIDs(i);
+            if (realitio.isFinalized(question_id)) {
```

Comment:
this change excludes questions with an answer that is not finalized yet, right? we need to count those too